templates/PlayFab_API.h.ejs

```diff
@@ -22,7 +22,7 @@ namespace PlayFab
 <% } %>
         // ------------ Generated API calls
 <% for (var callIdx = 0; callIdx < api.calls.length; callIdx++) { var apiCall = api.calls[callIdx];
-%>        static void <%- apiCall.name %>(<%- api.name %>Models::<%- apiCall.request %>& request, ProcessApiCallback<<%- api.name %>Models::<%- apiCall.result %>> callback, ErrorCallback errorCallback = nullptr, void* customData = nullptr);
+%>        static void <%- apiCall.name %>(<%- api.name %>Models::<%- apiCall.request %>& request, const ProcessApiCallback<<%- api.name %>Models::<%- apiCall.result %>> callback, const ErrorCallback errorCallback = nullptr, void* customData = nullptr);
```

Comment:
I didn't think these pass by value functions should be const

Reply:
Any reason they shouldn't be?
We won't modify their function pointers.